x/incentives/keeper/distribute.go

```diff
@@ -68,7 +75,8 @@ func (k Keeper) moveActiveGaugeToFinishedGauge(ctx sdk.Context, gauge types.Gaug
 	return nil
 }
 
-// GetLocksToDistribution get locks that are associated to a condition.
+// GetLocksToDistribution returns locks that match the provided lockuptypes QueryCondition AND have yet to be distributed to yet.
+// TODO: Why is this exported?
 func (k Keeper) GetLocksToDistribution(ctx sdk.Context, distrTo lockuptypes.QueryCondition) []lockuptypes.PeriodLock {
 	switch distrTo.LockQueryType {
 	case lockuptypes.ByDuration:
```

Comment:
Does anyone know why this is public? It's not used anywhere else and every other function here is private

Reply:
If that's the case, I think we should unexport it. Method/function access scope should be limited to where it is used. If there is a need for it to be used somewhere outside keeper, we can revisit it later